legal-api/tests/unit/resources/v2/test_business_filings/test_filing_documents.py

```diff
@@ -208,6 +209,20 @@ def test_unpaid_filing(session, client, jwt):
       },
      HTTPStatus.OK, '2017-10-01'
      ),
+    ('special_res_correction', 'CP7654321', Business.LegalTypes.COOP.value,
+     'correction', CORRECTION_CP_SPECIAL_RESOLUTION, None, None, Filing.Status.COMPLETED,
+     {'documents': {
+        'certifiedRules': f'{base_url}/api/v2/businesses/CP7654321/filings/1/documents/certifiedRules',
+        'correctedSpecialResolutionApplication': f'{base_url}/api/v2/businesses/CP7654321/filings/1/documents/correctedSpecialResolutionApplication' ,
+        'legalFilings': [
+            {'correction': f'{base_url}/api/v2/businesses/CP7654321/filings/1/documents/correction'},
```

Comment:
I'm not sure about the content of this. Should `correctedSpecialResolutionApplication` be included in there instead?

Reply:
![Uploading image.png…]()